CHANGES.md

```diff
@@ -1,3 +1,12 @@
+## 1.4.3 2018-06-12
+
+This is a follow up release that  includes diagnostic tracing for H2 requests. The feature was 
+scheduled for version 1.4.2 but did not make it in time for the release.
+
+Full release notes:
+
+* Enables third-party identifiers to emit router stats.  
```

Comment:
Consider using similar wording for this feature from a previous release:

`* Pass stack params to announcer plugins, allowing them to report metrics correctly.`